### DISCUSSION TITLE
Different creature behavior in Singleplayer and Multiplayer (Reverse)

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14719] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Creatures behave differently in SP and MP

Specifically in the case of attacks using `AfterAttack="ReverseUntilCanAttack"`, in singleplayer the creature will properly reverse, in multiplayer it will turn around. 
This might affect other aspects of creature behavior.

Examples with vanilla and modded creature:

https://github.com/user-attachments/assets/e05c7706-fb31-403e-bf18-9b1c3f390dea

https://github.com/user-attachments/assets/cffb4fde-bbdd-40a2-9313-81d1d3edc066

### Reproduction steps

Go in the abyss
Turn on sonar
Spawn latcher in SP
Host a game and do the same in MP

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly, fix coming in the next Unstable build. Closing discussion.
